sample/SampleServer/SampleServer.csproj

```diff
@@ -10,7 +10,7 @@
 
     <ItemGroup>
         <ProjectReference Include="../../src/Server/Server.csproj" />
-        <PackageReference Include="Microsoft.Extensions.Logging" VersionOverride="3.0.0" />
```

Comment:
The sample project was failing to restore because its dependencies updated to 6.0.0, but it was not.